build.sbt

```diff
@@ -1,7 +1,8 @@
 import sbtcrossproject.CrossPlugin.autoImport.{ crossProject, CrossType }
 
 val mainScala = "2.12.13"
-val allScala  = Seq("2.13.5", mainScala)
+val scala13   = "2.13.5"
```

Comment:
Can you remove this? It's not needed and it will conflict with my Scala 3 PR where I named it differently.